packages/server/modules/blobstorage/services/management.ts

```diff
@@ -39,3 +44,20 @@ export const uploadFileStreamFactory =
 
     return { blobId, fileName, fileHash }
   }
+
+export const getFileStreamFactory =
+  (deps: { getBlobMetadata: GetBlobMetadata }) =>
+  async <
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
```

Comment:
getting rid of any would require actual refactoring of this and some tests, which we want to avoid - some TS is better than no TS anyway!